workers/unity/Assets/Playground/Scripts/MonoBehaviours/FlashOnCollision.cs

```diff
@@ -0,0 +1,45 @@
+﻿using Generated.Playground;
+using Improbable.Gdk.Core.GameObjectRepresentation;
+using UnityEngine;
+
+public class FlashOnCollision : MonoBehaviour
+{
+    [Require] private Generated.Playground.Collisions.Reader reader;
+
+    private float collideTime;
+    private bool flashing = false;
+    private const float flashTime = 0.2f;
+
+    void OnEnable()
+    {
+        if (reader != null)
```

Comment:
You don't need to null check these. This is a guarantee the gdk provides.

Reply:
It will be once prefab preprocessing is in, that's what Firtina's working on at the moment. But for now there are some quick enable/disable calls on startup with null readers.